src/shared/ui/list/index.ts

```diff
@@ -0,0 +1,3 @@
+export { List } from './list';
+export { type ListData } from './list';
+export { type ListProps } from './list';
```

Comment:
Maybe we can write 
`export { List, type ListData, type ListProps } from './list';`

Reply:
Fixed